docs/DeveloperGuide.md

```diff
@@ -278,6 +287,8 @@ The following sequence diagram illustrates how the delete assignment operation w
 
 ### Edit Assignment Feature
 
+**Similar internship feature**: `Edit internship Role` feature and `Edit internship Task` feature
```

Comment:
maybe it is a good idea to say that it is similar to our EditInternshipTaskCommand and not EditInternshipRoleCommand because InternshipRole can have multiple fields edited